src/daemon/collectd.h

```diff
@@ -186,10 +186,6 @@
 #include <sys/param.h>
 #endif
 
-#if HAVE_KSTAT_H
```

Comment:
`"collectd.h"` used to be the header defining the global variables, so it should include `"globals.h"` now so that plugins continue working / compiling.

Reply:
Yes, it's included at the bottom :)